indico/modules/events/abstracts/templates/display/call_for_abstracts.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {%- trans %}Call for Abstracts{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="call-for-abstracts">
+        {% if cfa.is_open %}
+            <table class="dates">
+                <tr>
+                    <td><strong>{% trans %}Abstract submission opening day:{% endtrans %}</strong></td>
+                    <td>
+                        {%- trans date=cfa.start_dt|format_date(format="long", timezone=tz) -%}
+                            {{ date }}
+                        {%- endtrans -%}
+                    </td>
```

Comment:
We could use a `<time>` tag instead, similar to `abstracts/notification_log.html`:
``` html
<time datetime="{{ entry.sent_dt.isoformat() }}">
    {{- entry.sent_dt|format_human_date -}}
</time>
```

Also, the date doesn't need to be translated.